community/io/src/main/java/org/neo4j/io/pagecache/PageCache.java

```diff
@@ -94,4 +95,17 @@
 
     /** The max number of cached pages. */
     int maxCachedPages();
+
+    /**
+     * Move the file from the given source path, to the given target path.
+     *
+     * Both files have to be unmapped when performing the move, otherwise an exception will be thrown.
+     *
+     * @param sourceFile The name of the file to move.
+     * @param targetFile The new name of the file after the move.
+     * @param copyOptions Options to modify the behaviour of the move in possibly platform specific ways. In particular,
+     * {@link java.nio.file.StandardCopyOption#REPLACE_EXISTING} may be used to overwrite any existing file at the
+     * target path name, instead of throwing an exception.
+     */
+    void moveFile( File sourceFile, File targetFile, CopyOption... copyOptions ) throws IOException;
```

Comment:
i find it's really confusing that FS abstraction doing same functionality by method called 'renameFile' and here we define method 'moveFile'. i think those 2 should have same name.